example/package.json

```diff
@@ -3,49 +3,53 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios --list-devices",
+    "android": "react-native run-android --appId com.pagerviewexample",
+    "build:android": "npm run mkdist && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
+    "build:ios": "npm run mkdist && react-native bundle --entry-file index.js --platform ios --dev false --bundle-output dist/main.ios.jsbundle --assets-dest dist",
```

Comment:
You might want to add `--dev false` in `.github/workflows/ios.yml` instead.

Reply:
I think this script will be only used for building on CI, since for development `yarn ios` already fires up packager.